src/main/java/com/openlattice/socrates/Socrates.java

```diff
@@ -415,6 +415,7 @@ public static MultiLayerNetwork trainModel(
                 .layer( 2, new OutputLayer.Builder( LossFunction.NEGATIVELOGLIKELIHOOD )
                         .activation( Activation.SOFTMAX )
                         .nIn( numInputs ).nOut( outputNum ).build() )
+                .backpropType( BackpropType.Standard )
```

Comment:
Does order matter? Is it ok for `backpropType()` to come after the `layer()`s?

Reply:
Order does matter in that it appears to no longer be able to go anywhere except for after layer()s

Reply:
🤔

Reply:
i think .backpropType() will be meaningless without .backprop()?